presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -68,6 +71,14 @@ public void testGrantRevoke()
         assertThat(() -> bobExecutor.executeQuery(format("DELETE FROM %s WHERE day=3", tableName))).
                 failsWithMessage(format("Access Denied: Cannot delete from table default.%s", tableName));
 
+        // test SHOW GRANTS
```

Comment:
Also, please test SHOW GRANTS ON TABLE x and test that SHOW GRANTS ON USER <not-session-user> fails. Either here or via unit/integration tests.